src/utils/metrics.ts

```diff
@@ -46,20 +46,20 @@ export const blockHeight = new client.Gauge({
 });
 
 export const reorgDepth = new client.Gauge({
-  name: "watch_tower_reorg_depth",
-  help: "Depth of the current reorg",
+  name: "watch_tower_latest_reorg_depth",
+  help: "Depth of the last reorg",
   labelNames: ["chain_id"],
 });
 
-export const blockTime = new client.Gauge({
-  name: "watch_tower_block_time_seconds",
+export const blockProducingRate = new client.Gauge({
+  name: "watch_tower_block_producing_rate_seconds",
   help: "Time since last block",
   labelNames: ["chain_id"],
 });
 
 export const eventsProcessedTotal = new client.Counter({
-  name: "watch_tower_events_processed_total",
-  help: "Total number of events processed",
+  name: "watch_tower_order_creation_events_total",
+  help: "Total number of conditional order creation events processed",
   labelNames: ["chain_id"],
 });
```

Comment:
I'm not sure about this as with the case of `MerkleRoot`, when doing `setRoot`, it's undefined as to whether you are actually _creating_ more orders or not (as the size of the merkle tree may actually be smaller, ie. deleting/removing orders).

Reply:
@mfw78 good point, happy to change it to anything. I think just `events` its also to broad, there's for example https://github.com/cowprotocol/composable-cow/blob/main/src/ComposableCoW.sol#L46 as a event. 

What about `watch_tower_order_defintion_events_total`?

Reply:
https://github.com/cowprotocol/watch-tower/pull/111